src/js/main.js

```diff
@@ -4,19 +4,31 @@ import share from './lib/share';
 import sheetToDOM from './lib/sheettodom';
 import emailsignupURL from './lib/emailsignupURL';
 
-var shareFn = share('Interactive title', 'http://gu.com/p/URL', '#Interactive');
+// TODO remove once information is in sheet
+const tempSocialConfig = {
+    title: 'Interactive title',
+    url: 'http://gu.com/p/URL',
+    hashtag: '#Interactive',
+    facebookImage: 'https://media.guim.co.uk/4bf1338c1496f82606c7c0bc41274e22cfe96425/358_294_3280_1968/1000.jpg',
```

Comment:
I would be tempted to refactor the `share.js` lib, to make the image parameters optional so it just falls back to the `og:image` provided by Composer/Frontend. I think all this is really geared to where they want a custom share CTA which isn't needed for that project.
However, if that proves tricky, I don't mind the current approach.